Blog.API/Blog.Dependencies/Dependencies.cs

```diff
@@ -67,4 +75,16 @@ private static void ConfigureMediatR(this IServiceCollection services)
     {
         services.AddMediatR(cfg => cfg.RegisterServicesFromAssemblyContaining<GetPostsHandler>());
     }
+
+    private static void ConfigureValidators(this IServiceCollection services)
+    {
+        services.AddScoped<IValidator<CreateCategoryRequest>, CreateCategoryRequestValidator>();
+        services.AddScoped<IValidator<UpdateCategoryRequest>, UpdateCategoryRequestValidator>();
+
+        services.AddScoped<IValidator<CreateCommentRequest>, CreateCommentRequestValidator>();
+        services.AddScoped<IValidator<UpdateCommentRequest>, UpdateCommentRequestValidator>();
+
+        services.AddScoped<IValidator<CreatePostRequest>, CreatePostRequestValidator>();
+        services.AddScoped<IValidator<UpdatePostRequest>, UpdatePostRequestValidator>();
```

Comment:
There is no need to register every element manually. We can use `services.AddValidatorsFromAssembly(typeof(CreateCommentRequestValidator).Assembly);`